pkg/httputil/httputil.go

```diff
@@ -62,3 +62,13 @@ func IsFiltered(whiteList string, feedState model.FeedState) bool {
 	}
 	return whiteList == string(feedState)
 }
+
+// IsContainNonASCII return true if a string contain non-ascii char
+func IsContainNonASCII(s string) bool {
+	for _, r := range s {
+		if r < 0 || r > 127 {
```

Comment:
Maybe we can use unicode.MaxASCII?

Also, will this value be less than 0?

Reply:
good
it seems will not less than 0.